test/e2e/performanceprofile/functests/utils/nodes/nodes.go

```diff
@@ -46,8 +46,9 @@ type NumaNodes struct {
 
 // NodeCPU Structure
 type NodeCPU struct {
-	CPU  string `json:"cpu"`
 	Node string `json:"node"`
+	Core string `json:"core"`
+	CPU  string `json:"cpu"`
```

Comment:
for the future: can these be `int`s?

Reply:
It can't be int's because the lscpu -e=node,core,cpu -J returns json output in strings 

example:

`
{
   "cpus": [
      {"node": "0", "core": "0", "cpu": "0"},
      {"node": "1", "core": "1", "cpu": "1"},
      {"node": "0", "core": "2", "cpu": "2"},
      {"node": "1", "core": "3", "cpu": "3"},
      {"node": "0", "core": "4", "cpu": "4"},
      {"node": "1", "core": "5", "cpu": "5"},
      {"node": "0", "core": "6", "cpu": "6"},
      {"node": "1", "core": "7", "cpu": "7"},
      {"node": "0", "core": "8", "cpu": "8"},

`

Reply:
The full output of lscp -e=node,core,cpu can be found here: https://paste.centos.org/view/83ec74d8